tornjak-frontend/src/components/cluster-management.tsx

```diff
@@ -77,12 +115,15 @@ class ClusterManagement extends Component {
     });
   }
 
-  prepareAgentsList() {
+  prepareAgentsList(): void {
     var prefix = "spiffe://";
-    let localAgentsIdList = [];
+    let localAgentsIdList: { label: string; }[] = [];
```

Comment:
yea - definitely would help to define the type, this is repeated so often

Reply:
Done!